ega-data-api-filedatabase/src/main/resources/application.yml

```diff
@@ -1,20 +1,38 @@
-spring.profiles: no-oss
+spring.profiles.active: no-oss
```

Comment:
I think you added active by mistake. Maybe it would be clearer to rename the file as application-no-oss.yml so it is easier to see that this file is only loaded on the no-oss profile

Reply:
Fixed.